.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "jquery": true,
+        "jest": true
```

Comment:
I am interested to know why you switched from Jasmine to Jest

Reply:
it is easy to include the coverage, also I had problem of the Jasmine web interface to include code modules